programs/virtual-curve/src/instructions/migration/create_locker.rs

```diff
@@ -0,0 +1,154 @@
+use crate::{
+    constants::seeds::{BASE_LOCKER_PREFIX, POOL_AUTHORITY_PREFIX},
+    state::{MigrationProgress, PoolConfig, VirtualPool},
+    *,
+};
+use anchor_lang::solana_program::{program::invoke, system_instruction};
+use anchor_spl::{
+    token_2022::{mint_to, MintTo},
+    token_interface::{TokenAccount, TokenInterface},
+};
+use locker::cpi::accounts::CreateVestingEscrowV2;
+
+#[derive(Accounts)]
+pub struct CreateLockerCtx<'info> {
+    /// Config
+    #[account(mut, has_one = config, has_one = creator)]
```

Comment:
We need to make sure the `base_vault` belongs to the `virtual_pool` by `has_one = base_vault` since the `base_vault` authority is not `virtual_pool`

Reply:
Fixed